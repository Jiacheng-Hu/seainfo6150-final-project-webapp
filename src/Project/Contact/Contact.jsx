import React from "react";

const Contact = () => {

    return (
        
        <div className="text-center">
            <h2>Contact Us</h2>
            <div className="contactUs">
                <div className="labels">Email Address:</div>
                <div className="margin-left">hu.jiac@neu.edu</div>
            </div>
            <div  className="contactUs margin-top">
                <div className="labels">Address:</div>
                <div className="margin-left">5346 15th Ave NE</div>
                <div>Seattle, WA 98105</div>
            </div>
            <div  className="contactUs margin-top">
                <div className="labels">Phone Number:</div>
                <div className="margin-left">206-123-4321</div>
            </div>
        </div>
    )
}

export default Contact;