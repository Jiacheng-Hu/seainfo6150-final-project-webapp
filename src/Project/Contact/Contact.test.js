import React from "react";
import Contact from "./Contact";

describe("Contact tests", () => {
    it("renders correctly", () => {
        const {container} = render(<Contact/>);
        expect(container).toMatchSnapshot();
    });
});