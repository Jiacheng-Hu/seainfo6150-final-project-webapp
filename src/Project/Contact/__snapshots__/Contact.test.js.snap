// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact tests renders correctly 1`] = `
<div>
  <div
    class="text-center"
  >
    <h2>
      Contact Us
    </h2>
    <div
      class="contactUs"
    >
      <div
        class="labels"
      >
        Email Address:
      </div>
      <div
        class="margin-left"
      >
        hu.jiac@neu.edu
      </div>
    </div>
    <div
      class="contactUs margin-top"
    >
      <div
        class="labels"
      >
        Address:
      </div>
      <div
        class="margin-left"
      >
        5346 15th Ave NE
      </div>
      <div>
        Seattle, WA 98105
      </div>
    </div>
    <div
      class="contactUs margin-top"
    >
      <div
        class="labels"
      >
        Phone Number:
      </div>
      <div
        class="margin-left"
      >
        206-123-4321
      </div>
    </div>
  </div>
</div>
`;
